python/xorbits/_mars/dataframe/datasource/utils.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2022-2023 XProbe Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from ...lib.filesystem import LocalFileSystem, get_fs
+
+
+def convert_to_abspath(path: str, storage_options: dict = None):
```

Comment:
According to your implementation, the type of parameter "path" can be list, tuple, or string. Therefore, please provide the complete type annotation.

Reply:
Also, the type annotation for parameter "storage_options" should be "dict | None"

Reply:
Ok